src/Microsoft.ML.StaticPipe/LbfgsStatic.cs

```diff
@@ -9,15 +9,15 @@
 
 namespace Microsoft.ML.StaticPipe
 {
-    using Options = LogisticRegressionBinaryTrainer.Options;
+    using Options = LbfgsLogisticRegressionTrainer.Options;
 
     /// <summary>
     /// Binary Classification trainer estimators.
     /// </summary>
-    public static class LbfgsBinaryClassificationStaticExtensions
+    public static class LbfgsBinaryExtensions
```

Comment:
>LbfgsBinaryExtensions [](start = 24, length = 21)

I think these are just static extensions, if possible could you keep static in the name of the class?